project/build.scala

```diff
@@ -0,0 +1,83 @@
+import sbt.Keys._
+import sbt._
+
```

Comment:
This is because the need of nested sub-projects and following good practices:
"An sbt build definition can contain files ending in .sbt, located in the base directory, and files ending in .scala, located in the project subdirectory of the base directory.

You can use either one exclusively, or use both. A good approach is to use .sbt files for most purposes, and use .scala files only to contain what can't be done in .sbt:

    to customize sbt (add new settings or tasks)
    to define nested sub-projects"

From: http://www.scala-sbt.org/0.12.2/docs/Getting-Started/Basic-Def.html

More info: http://www.scala-sbt.org/0.12.2/docs/Getting-Started/Multi-Project.html